conda_build/inspect_pkg.py

```diff
@@ -110,23 +110,21 @@ def __str__(self):
 untracked_package = _untracked_package()
 
 
+@deprecated.argument("3.28.0", "4.0.0", "platform", rename="subdir")
+@deprecated.argument("3.28.0", "4.0.0", "prepend")
+@deprecated.argument("3.28.0", "4.0.0", "minimal_hint")
```

Comment:
four dot oh, four dot oh!

Reply:
not for long https://github.com/conda/conda-build/issues/5064

Reply:
oh no! i forgot to update the deprecation versions!

```suggestion
@deprecated.argument("24.1.0", "24.3.0", "platform", rename="subdir")
@deprecated.argument("24.1.0", "24.3.0", "prepend")
@deprecated.argument("24.1.0", "24.3.0", "minimal_hint")
```